### PR TITLE
Introduces DocHeader tool to check header docs

### PR DESCRIPTION
**Pull request by froschdesign**

### CODE REVIEW

**Comment by Ocramius** [comment]
@froschdesign can we have this against `develop` please?

File: .docheader [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/**
+ * @see       https://github.com/zendframework/zend-db for the canonical source repository
+ * @copyright Copyright (c) %regexp:(20\d{2}-)20\d{2}% Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-db/blob/master/LICENSE.md New BSD License
+ */
```
Comment (webimpress) [inline .docheader]: Please add empty line at the end of the file.

File: composer.json [hunk 2]
```diff
@@ -44,5 +45,8 @@
         "psr-4": {
             "ZendTest\\Db\\": "test/"
         }
+    },
+    "scripts": {
+        "license-check": "vendor/bin/docheader check src/"
```
Comment (webimpress) [inline composer.json]: Shouldn't we check it also in `test/` directory?

**Comment by froschdesign** [comment]
No longer relevant.

**Pull request closed without merging**